pkg/apis/cr/v1alpha1/types.go

```diff
@@ -255,11 +296,16 @@ const (
 
 // Location
 type Location struct {
-	Type     LocationType `json:"type"`
-	Bucket   string       `json:"bucket"`
-	Endpoint string       `json:"endpoint"`
-	Prefix   string       `json:"prefix"`
-	Region   string       `json:"region"`
+	// Type specifies the kind of object storage that would be used to upload the backup objects. Currently supported values are: "GCS", "S3Compliant", "Azure" and "Kopia".
```

Comment:
```suggestion
	// Type specifies the kind of object storage that would be used to upload the
	// backup objects. Currently supported values are: "GCS", "S3Compliant",
	// and "Azure".
```
Let's not add `Kopia`